### DISCUSSION TITLE
500 error when surfing firefly

### DISCUSSION THREAD

**Discussion by mamob07** [firefly-iii/firefly-iii#9510] [Firefly III]
Hello I get a 500 internal server error when surfing to any page of firefly, also /debug page. It says this: 
`Allowed memory size of 134217728 bytes exhausted (tried to allocate 25832752 bytes)`
The strange fact is that I have 128M as php memory_limit and he tries to allocate less memory. I tried of course to increase the size of memory_limit in php.ini to 256M but the error is basically the same: 
`Allowed memory size of 268435456 bytes exhausted (tried to allocate 125829152 bytes)`

**Comment by JC5** [comment]
I'm not sure. What happens when you keep increasing the memory limit? When does it stop complaining?

**Reply by mamob07** [reply]
With 1024M I get this: 
`The stream or file "/www/wwwroot/<website path>/storage/logs/ff3-fpm-fcgi-2024-11-30.log" could not be opened in append mode: Failed to open stream: Permission denied`

**Reply by JC5** [reply]
We're getting somewhere!

https://docs.firefly-iii.org/references/faq/docker/#i-get-failed-to-open-stream-permission-denied-on-log-files
